ui/perfherder/compare/CompareTable.jsx

```diff
@@ -28,14 +30,44 @@ export default class CompareTable extends React.PureComponent {
       displayNumber(percentage),
     )}% ${improvement ? 'better' : 'worse'})`;
 
+  // humane readable signature name
+  getSignatureName = (testName, platformName) =>
+    [testName, platformName].filter(item => item !== null).join(' ');
+
+  getHashBasedId = (testName, platformName = null) => {
+    const { hashFunction } = this.props;
+
+    const tableSection = platformName === null ? 'header' : 'row';
+    const hashValue = hashFunction(
+      this.getSignatureName(testName, platformName),
+    );
+
+    return `comptable-${tableSection}-${hashValue}`;
+  };
+
   render() {
-    const { data, testName } = this.props;
+    const { data, onPermalinkClick, testName } = this.props;
+
     return (
-      <Table sz="small" className="compare-table mb-0 px-0" key={testName}>
+      <Table
+        id={this.getHashBasedId(testName)}
+        aria-label="Comparison table"
+        sz="small"
+        className="compare-table mb-0 px-0"
+        key={testName}
+      >
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
               <span>{testName}</span>
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
I think there might be acceptable alternative options listed in the `jsx-a11y/anchor-is-valid` rule docs. Since we're making an effort to improve accessibility with Perfherder, I'd rather not disable rules like these unless there's a good reason to.

Reply:
Yep, fixed that!